kms/tpmkms/tpmkms.go

```diff
@@ -174,6 +180,34 @@ func New(_ context.Context, opts apiv1.Options) (kms *TPMKMS, err error) {
 			}
 			kms.identityRenewalPeriodPercentage = *percentage
 		}
+
+		// opt-in for enabling CAPI integration on Windows for certificate
+		// management. This will result in certificates being stored to or
+		// retrieved from the Windows certificate stores.
+		if runtime.GOOS == "windows" {
+			if withCAPI := u.GetBool("with-capi"); withCAPI {
+				fn, ok := apiv1.LoadKeyManagerNewFunc(apiv1.CAPIKMS)
+				if !ok {
+					return nil, fmt.Errorf("CAPIKMS not compiled") // TODO(hs): better error message; see capi package
+				}
+				km, err := fn(ctx, apiv1.Options{
+					Type: apiv1.CAPIKMS,
+					URI:  "capi:provider=Microsoft Platform Crypto Provider", // TODO(hs): more (default) CAPI properties?
```

Comment:
Made it a constant in 79a78746151be271ee090884a0e6bdac4ecdf925. Moving it into a shared package won't help much, and might introduce a hard dependency on the `capikms` here, which I want to prevent.